guides/common/attributes-foreman-el.adoc

```diff
@@ -2,4 +2,8 @@
 :foreman-installer-package: foreman-installer
 :installer-scenario-smartproxy: foreman-installer --no-enable-foreman
 :installer-scenario: foreman-installer --scenario foreman
-:project-installation-guide-title: Installing Foreman {ProjectVersion} server on Enterprise Linux
+
+// Overrides for titles
+:OSBase: RHEL/CentOS
```

Comment:
```suggestion
:OSBase: EL
```

I am unsure if it makes sense to list "CentOS" here. Do you mean CentOS 7? Or CentOS 8 Stream? Maybe just `EL` then?

Reply:
@maximiliankolb Are you absolutely sure that changing this is a good idea?

Reply:
@melcorr Melanie, can we have a second opinion on this, please?

Reply:
Can you please elaborate a bit @melcorr ? Do you think we should not override this attribute at all?

I'd argue that a) EL captures all different OS best (at least upstream) and b) that there is a huge difference in RHEL and CentOS since it became CentOS Stream. I thought "EL" is the easiest solution.

I have no strong feeling either way.

Reply:
Oki, so I will remove the OSBase attribute and use plain text instead as it was before. I hope I'm understanding you correctly on this.